hack/generate-crds.sh

```diff
@@ -105,7 +105,7 @@ generate_group () {
     # etcd-druid, due to adverse interaction with VPA.
     # See https://github.com/gardener/gardener/pull/6850 and https://github.com/gardener/gardener/pull/8560#discussion_r1347470394
     # TODO(shreyas-s-rao): Remove this workaround as soon as the scale subresource is supported properly.
-    etcd_api_types_file="$(dirname "$0")/../vendor/github.com/gardener/etcd-druid/api/v1alpha1/types_etcd.go"
+    etcd_api_types_file="${package_path%;}/types_etcd.go"
```

Comment:
Hm, why the substitution?

Reply:
You mean the `%;`? 
```
  local package_path="$(go list -f '{{ .Dir }}' "$package" | tr '\n' ';')"
```
This commands replaces new line with `;`, so I am removing it for the path.